astroid/brain/brain_stdlib.py

```diff
@@ -437,3 +446,4 @@ def shutdown(self):
 register_module_extender(MANAGER, 'multiprocessing.managers',
                          multiprocessing_managers_transform)
 register_module_extender(MANAGER, 'multiprocessing', multiprocessing_transform)
+register_module_extender(MANAGER, 'threading', thread_transform)
```

Comment:
Did you see my comment from https://bitbucket.org/logilab/astroid/pull-requests/98/add-support-for-inference-on-threadinglock/diff regarding transforms vs inference tip?